content/components/selectpanel.mdx

```diff
@@ -125,24 +433,64 @@ Select panel should be anchored to its trigger.
     />
-    <Caption>Selecting the submit button or pressing the enter key while focused in the listbox should submit any user input.</Caption>
+    <Caption>
+      Selecting the submit button or pressing the enter key while focused in the listbox should submit any user input.
+    </Caption>
   </div>
 </Box>
 
 ### Responsive
 
-Select panel can become a full screen dialog or a bottom sheet at narrow viewports. Support at 320px is required if the select panel is larger at wider viewports. See [narrow-viewport placement](https://primer.style/design/components/dialog#narrow-viewport-placement) in the Dialog guidelines for more details.
+On smaller viewports, the select panel can be transformed into either a full-screen dialog or a bottom sheet dialog. To ensure usability on touch devices, the confirmation buttons and action list items are also increased to a medium size, ensuring a sufficiently large hit target.
+
+When adding an extra action in the footer, it is important to ensure that it still fits within a 320px viewport.
+
+<img
+  width="960"
+  alt="mobile"
+  src="https://github.com/primer/design/assets/980622/ae6d19c2-7bfa-4357-be65-341c0728b9a8"
+/>
+
+See the [Dialog guidelines](https://primer.style/design/components/dialog) for more details.
+
+## Cancel/submit
+
+In most scenarios, we recommend using declarative controls, except for some single-select instances as [demonstrated earlier](#list-single-select).
+However, we acknowledge that in specific areas of the product that users interact with frequently throughout the day, employing declarative controls may create the perception of a sluggish experience.
+For example, if the issue creation process involves too many steps, it may discourage customers from reporting issues.
+
+We are currently [conducting internal experiments](https://github.com/github/platform-ux/issues/1327) to validate those considerations, while closely collaborating with our accessibility auditors to ensure the optimal experience for all users.
+
+Our current position on ommiting declaritive buttons takes in account a mix of considerations:
+
+- **Reversibility**: If the selection has the potential to initiate workflows or send notifications that cannot be easily reversed, then it is not appropriate to make an exception.
+- **Passive selections**: Exceptions are more warranted when dealing with selections that are part of a larger form and are not submitted until the user presses the "Send" button. In such cases, it is crucial to ensure that upon closing the select panel, the focus is returned to the trigger element and that this indicates the selection. Passive selections can also be defined as selections that do not directly modify the database.
+- **Frequency of use**: Exceptions are less warranted for settings pages since they are not frequently used. On the other hand, creating an issue is a task that can occur multiple times a day, making an exception more justified in this case.
```

Comment:
I don't agree with this one either. We shouldn't be choosing when to implement accessible patterns based on frequency of use. On the contrary, highly visible and highly used features should have the most accessibility built in and should not have exceptions.

Reply:
Removing this section to push this pull request through.